chore(docs): add a contributing guide (#1815)
DevExpress/devextreme-reactive: CONTRIBUTING.md

# Contribution Guidelines

## Issues and Feature Requests

> If you got a question, bug report, or feature request, submit an issue to [our repository](https://github.com/DevExpress/devextreme-reactive) but please check the [list of active issues](https://github.com/DevExpress/devextreme-reactive/issues) first to avoid duplicates. 

Reviewer: "submit an issue" seems like you are submitting a problem, something negative. Perhaps 'submit a ticket'?

Reviewer 2: There is no such thing as a ticket on GitHub, only issues and pull requests

Reviewer 2: ```suggestion
If you got a question, bug report, or feature request, submit an issue to [our repository](https://github.com/DevExpress/devextreme-reactive). You should check the [list of active issues](https://github.com/DevExpress/devextreme-reactive/issues) first to avoid duplicates. 
```

Reviewer: Ok. If you got a question, bug report, or feature request, submit an issue to [our repository](https://github.com/DevExpress/devextreme-reactive). You should check the [list of active issues](https://github.com/DevExpress/devextreme-reactive/issues) first to avoid duplicates.

> If you got a question, bug report, or feature request, submit an issue to [our repository](https://github.com/DevExpress/devextreme-reactive) but please check the [list of active issues](https://github.com/DevExpress/devextreme-reactive/issues) first to avoid duplicates. 

Reviewer: ```suggestion
If you got a question, bug report, or feature request, submit a ticket to [our repository](https://github.com/DevExpress/devextreme-reactive). You should check the [list of active issues](https://github.com/DevExpress/devextreme-reactive/issues) first to avoid duplicates. 
```

> If you own an active DevExtreme license, feel free to contact us in the [Support Center](https://www.devexpress.com/ask) to receive an answer in shorter time.

Reviewer: ```suggestion
You can use the [Support Center](https://www.devexpress.com/ask) to contact us if you have an active DevExtreme license.
```

Reviewer 2: The fact that we answer quicker to tickets in the Support Center is important

Reviewer: If you have an active DevExtreme license, contact us in the [Support Center](https://www.devexpress.com/ask) for an expedited response.

> Got a Question about our components? At first, check our [documentation](https://devexpress.github.io/devextreme-reactive/) and existing [issues](https://github.com/DevExpress/devextreme-reactive/issues) in our repository. If you didn't find any useful information, create a new issue with your question. If you own an active DevExtreme license, feel free to contact us in the Support Center (https://www.devexpress.com/ask) to receive an answer shortly.

Reviewer 3: Add link to SC with next syntax `[Support Center](ulr)`

## Contribute Code and Content

Important: **Read Section 5 of the [LICENSE AGREEMENT](LICENSE.md#5-submission-of-contributions) before you submit any contribution.**

> Please keep in mind that DevExtreme Reactive has a development schedule; we are unable to accept every pull request from non-DevExpress contributors.

Reviewer: ```suggestion
Note that DevExtreme Reactive has a development schedule and we cannot accept every pull request from non-DevExpress contributors.
```

> However, you are welcome to fork the repository, modify the source code in your fork, and make your builds.

Reviewer: ```suggestion
However, you can fork the repository, modify the source code in your fork, and make your builds.
```
